CMakeLists.txt

```
@@ -2498,7 +2470,11 @@ set(CPACK_PACKAGE_INSTALL_DIRECTORY "Mixxx")
 set(CPACK_PACKAGE_EXECUTABLES "mixxx;Mixxx")
 set(CPACK_PACKAGE_ICON "${CMAKE_SOURCE_DIR}/res/images/mixxx_install_logo.bmp")
 set(CPACK_PACKAGE_HOMEPAGE_URL "https://www.mixxx.org/")
-set(CPACK_PACKAGE_FILE_NAME "Mixxx-${GIT_BRANCH}-r${GIT_COMMIT_COUNT}-${GIT_COMMIT_HASH}")
+if(GIT_COMMIT_DESCRIPTION STREQUAL "unknown-commit")
+  set(CPACK_PACKAGE_FILE_NAME "Mixxx-${MIXXX_VERSION}")
```

```suggestion
  set(CPACK_PACKAGE_FILE_NAME "mixxx_${MIXXX_VERSION}")
```
For debian compatibility and other Linux package files. Otherwise we need to special case that individual.